### ISSUE TITLE
How to Prepare the Inpainting Training Dataset

### ISSUE THREAD

**Issue by Bai426** [yangxy/GPEN#164]
Great Work! 
I am wondering whether you can specify how to prepare the training data for inpainting? For example, do I need to overlay the mask together? Where should I put the GT, noised data (input) and mask images? Your training codes seem to be based on stylgan2-pytorch, but I really want to know how you coordinate these inputs and ground truth together for the inpainting training.

Thanks!
